### PR TITLE
Don't visit indexes

### PR DESCRIPTION
**Pull request by bouk**
This mirrors the behavior of sqlalchemy_vertica https://github.com/jamescasbon/vertica-sqlalchemy/blob/43ee1fa57d452b5fe3240e4754f1f2770da16fdc/sqlalchemy_vertica/base.py#L179-L181

### CODE REVIEW

**Comment by graingert** [comment]
looks good. Change log?

**Comment by bouk** [comment]
Good call on the tests, turns out the vertica one has a bug, the visit_create_index method needs to be on DDLCompiler :smile: 

:shipit:

File: CHANGES.rst [hunk 1]
```diff
@@ -2,6 +2,9 @@
 0.1.3 (unreleased)
 ------------------
 
+- Don't visit indexes as Redshift doesn't support them
```
Comment (graingert) [inline CHANGES.rst]: .

**Comment by graingert** [comment]
@bouk okay, make sure to fix it in Vertica too!

**Comment by bouk** [comment]
k fixed the changelog

**Pull request merged**